src/components/WorkOrders/JobView.js

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from 'react'
+import JobsTable from './JobsTable'
+import { getRepairs } from '../../utils/frontend-api-client/repairs'
+import Spinner from '../Spinner/Spinner'
+import ErrorMessage from '../Errors/ErrorMessage/ErrorMessage'
+import { convertDate } from '../../utils/date'
+
+const JobView = () => {
+  const [workOrders, setWorkOrders] = useState([])
+  const [error, setError] = useState()
+  const [loading, setLoading] = useState(false)
+
+  const workOrderView = async () => {
+    setError(null)
+
+    try {
+      const data = await getRepairs()
+
+      data.forEach((job) => {
+        job.dateRaised = convertDate(job.dateRaised)
+        job.lastUpdated = convertDate(job.lastUpdated)
+      })
+
+      const sortedByDate = data.sort((a, b) => b.dateRaised - a.dateRaised)
```

Comment:
would be great to move this functionality of sorting data by date into `utils/date` and a test for this